### DISCUSSION TITLE
SMS provider other than Twilio

### DISCUSSION THREAD

**Discussion by fikrikarim** [supabase/supabase#3048] [Questions] [answered]
How hard is it to add sms provider other than Twilio? Right now Twilio is very expensive in my country, and the alternative is ~3x cheaper.

**Comment by kangmingtay** [comment, marked as answer]
Hey @fikrikarim, we are open to adding other sms providers besides Twilio. May I know which other provider you are considering? You can also create an issue on our auth repo (https://github.com/supabase/gotrue) and feel free to make a contribution to add a new sms provider too!!

**Reply by fikrikarim** [reply]
Right now I'm considering https://www.gosmsgateway.com/, as it's one of the most popular sms provider in Indonesia. I'm still thinking whether to just use OAuth or phone auth right now. If we decide to use phone auth, I might take a stab to make a contribution there. Thanks a lot 👍
